INSTALL.md

```diff
@@ -174,20 +174,12 @@ psql -d openstreetmap -c "CREATE EXTENSION btree_gist"
 
 ### PostgreSQL Functions
 
-We need to install special functions into the PostgreSQL databases, and these are provided by a library that needs compiling first.
+There are special database functions required by a (little-used) API call, the migrations and diff replication. The former two are provided as *either* pure SQL functions or a compiled shared library. It is recommended that you start with the pure SQL versions, as described below, and only install the compiled shared library if you are running a production server making a lot of `/changes` API calls or need the diff replication functionality.
```

Comment:
I would suggest rephrasing this to avoid the user from having to read, understand, or make a decision at this point. So for example, I think it's enough to say:

"We need to install some special functions into the PostgreSQL databases"

... and leave it at that. The explanation of the advantages of the compiled shared library can then live in the later section of the docs, and if someone gets that far and wants to use them, it's only a total of 3 lines of code to copy+paste (1 here for installing, 2 later for removing) so no big deal.

Reply:
Good points, thanks. I've made those changes in 2423d86. Please let me know what you think.

Reply:
Perfect